Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.7-slim
+
+RUN apt update && apt upgrade -y && \
+    apt install -y \
+        libmagic1 \
+        build-essential
+
+WORKDIR /bot
+
+COPY *.py /bot/
+COPY *.cfg /bot/
+COPY *.txt /bot/
+COPY eno /bot/eno/
+COPY debs /bot/debs/
```

Comment:
debs doesn't exist, instead of downloading the debs I would suggest. 

```
RUN git clone https://gitlab.matrix.org/matrix-org/olm.git /tmp/libolm \
    && cd /tmp/libolm \
    && make install
```

otherwise you should probably just `RUN wget` the debs before dpkg inside the Dockerfile